webpack.config.js

```diff
@@ -82,7 +82,7 @@ module.exports = {
             }
         },
         // Don't produce production output when a build error occurs.
-        noEmitOnErrors: prod
+        emitOnErrors: prod
```

Comment:
I believe this needs to be flipped now.
```suggestion
        emitOnErrors: !prod
```